Directory.Build.targets

```diff
@@ -78,4 +78,21 @@
     </KnownFrameworkReference>
   </ItemGroup>
 
+  <PropertyGroup>
+    <CoreCompileDependsOn>$(CoreCompileDependsOn);RemoveCodeStyleAnalyzers</CoreCompileDependsOn>
+  </PropertyGroup>
+
+  <!--
+    CodeStyle analyzers inserted in the SDK might not be compatible with the current public version of Visual Studio,
+    which causes failures when building from the IDE.
+  -->
```

Comment:
need a bug tracking this so we don't forget to undo

Reply:
Added